### DISCUSSION TITLE
How to Contribute?

### DISCUSSION THREAD

**Discussion by oliyan** [osm-in/openstreetmap.in#78] [General]
Please provide information about contributing guidelines for this project.

**Comment by planemad** [comment]
Hi @oliyan , many thanks for your interest to contribute. We don't have any published guidelines as such since very minimal work has gone into the site thus far.

Most of it was built by me over a few days many years ago and has not changed much since then. Feel free to open a PR for any change and I can take a look.

**Comment by oliyan** [comment]
Hi @planemad , I am a mainframe developer interested in open source. I don't have any idea as to how to contribute to this project, but I would like to contribute in any way possible. 

So if you have something in your mind for this project that needs another mind to contribute, please let me know.
